Database/Stored Procedures/Add_or_Update/add_visitor_visited_performance.sql

```diff
@@ -0,0 +1,52 @@
+/*==============================================================*/
+/* DBMS name:		FestiBase									*/
+/* PDM version:		6											*/
+/* Last edited:		08-01-2018									*/
+/* Edited by:		Mariusz Blautzik							*/
+/* Procedure:		Insert + Update VISITOR VISITED PERFORMANCE	*/
+/*==============================================================*/
+
+USE FestiBase
+GO
+
+/*
+	Procedure to add or update a BOUGHT_TICKET
+*/
+DROP PROC IF EXISTS sp_add_visitor_visited_performance
+GO
+CREATE PROC sp_add_visitor_visited_performance
+	@visitor_number				INT,
+    @performance_number			INT,
+	@insert						BIT
```

Comment:
dit is een beetje nutteloos sinds dit bedoeld was om een insert en update in een sp te krijgen, maar sinds dit alleen een insert is kan dit wel weg.